hooks/pre-checkout

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+export_canva_origin_url() {
+  echo "$(date '+[%Y-%m-%d %H:%M:%S]') INFO: starting pre-checkout"
+  local origin_url
+  local s3_path
+  if [[ -n "${BUILDKITE_PLUGIN_GITHUB_FETCH_SSM_PARAM:-}" ]] && [[ -n "${BUILDKITE_PLUGIN_GITHUB_FETCH_S3_URL:-}" ]]; then
```

Comment:
sorry I meant add the url form the param to the `BUILDKITE_PLUGIN_GITHUB_FETCH_S3_URL` env var.
I think we should still get the url from SSM even if we are not getting the checkout from S3.

Reply:
i think I addressed this comment, let me know if I didn't